subo/command/clean.go

```diff
@@ -0,0 +1,64 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/suborbital/subo/subo/context"
+)
+
+//CleanCmd  removes all of the target/.build folders for Runnables and deletes the .wasm files.
+func CleanCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clean",
+		Short: "remove build folders and .wasm files",
+		Long:  "remove all of target/.build folders and deletes .wasm files",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cwd, err := os.Getwd()
+			if err != nil {
+				cwd = "$HOME"
+			}
+
+			bctx, err := context.CurrentBuildContext(cwd)
+			if err != nil {
+				return errors.Wrap(err, "failed to get CurrentBuildContext")
+			}
+
+			if len(bctx.Runnables) == 0 {
+				return errors.New("🚫 no runnables found in current directory (no .runnable yaml files found)")
+			}
+			logStart(fmt.Sprintf("cleaning in %s", bctx.Cwd))
+
+			//regex to find .wasm files
+			fileRegEx, _ := regexp.Compile("^.+\\.wasm")
+
+			for i := 0; i < len(bctx.Runnables); i++ {
```

Comment:
You can do `for _, r := range bctx.Runnables` instead, and for each iteration of the loop, `r` will be one of the elements in the array.